internal/controller/monitoringconfig_controller.go

```diff
@@ -153,17 +153,21 @@ func (r *MonitoringConfigReconciler) reconcileVMAgent(ctx context.Context) error
 func (r *MonitoringConfigReconciler) genVMAgentSpec(ctx context.Context, monitoringConfigList *everestv1alpha1.MonitoringConfigList) (vmv1beta1.VMAgentSpec, error) {
 	spec := vmv1beta1.VMAgentSpec{
 		SelectAllByDefault: true,
-		ExtraArgs: map[string]string{
-			"memory.allowedPercent": "40",
-		},
-		Resources: corev1.ResourceRequirements{
-			Requests: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("250m"),
-				corev1.ResourceMemory: resource.MustParse("350Mi"),
+		CommonApplicationDeploymentParams: vmv1beta1.CommonApplicationDeploymentParams{
+			ExtraArgs: map[string]string{
+				"memory.allowedPercent": "40",
```

Comment:
Just curious why 40%? Is that recommended somewhere?

Reply:
I have no idea :sweat_smile: I didn't want to change it.

Reply:
Oh, I see, it was like this before😆 For some reason I thought it's a new parameter, that's why I got confused why this exact value is used. I see it was like this since [pmm dbaas](https://github.com/percona/dbaas-controller/blob/9900837a95c73b73eec61bd3b0198a106d2855c6/service/k8sclient/k8sclient.go#L1898-L1900). Never mind then.